products/firefox_android/adding_singlelocale.md

```diff
@@ -66,25 +66,25 @@ popping bug123456.patch
 patch queue now empty
 ```
 
-The patch is stored inside the `.hg/patches` folder in the root of the repository (in the suggested setup, the full path would be `~/mozilla/mercurial/mozilla-aurora/.hg/patches`). You can copy the file through the command line or the file explorer. For example, on Mac you can open the folder in Finder by typing:
+The patch is stored inside the `.hg/patches` folder in the root of the repository (in the suggested setup, the full path would be `~/mozilla/mercurial/mozilla-central/.hg/patches`). You can copy the file through the command line or the file explorer. For example, on Mac you can open the folder in Finder by typing:
 
 ```BASH
-$ open ~/mozilla/mercurial/mozilla-aurora/.hg/patches
+$ open ~/mozilla/mercurial/mozilla-unified/.hg/patches
 ```
 
 Or you can copy the file on the Desktop with
 
 ```BASH
-$ cp ~/mozilla/mercurial/mozilla-aurora/.hg/patches/bug123456.patch ~/Desktop
+$ cp ~/mozilla/mercurial/mozilla-unified/.hg/patches/bug123456.patch ~/Desktop
 ```
 
 Now you need to attach the file to Bugzilla and set an appropriate reviewer for it.
 
 ## After the patch has been reviewed
 
-Once the patch has been reviewed, you need to ask for approval: go in Details for the attachment, set `?` in the `approval‑mozilla‑aurora` flag and specify in the comment that the patch only needs to land in mozilla-aurora.
+Once the patch has been reviewed, you need to ask for approval: go in Details for the attachment, set `?` in the `approval‑mozilla‑central` flag and specify in the comment that the patch only needs to land in mozilla-central.
 
-Once approved for Aurora, sheriffs will land the patch. A couple of days after check on FTP if builds are being generated for this new locale: https://ftp.mozilla.org/pub/mobile/nightly/latest-mozilla-aurora-android-api-15-l10n/
+Once approved for Central, sheriffs will land the patch. A couple of days after check on FTP if builds are being generated for this new locale: http://ftp.mozilla.org/pub/mobile/nightly/latest-mozilla-central-android-api-15-l10n/
```

Comment:
Once reviewed, sheriffs or the reviewer will land the patch.

For Central I can land your patches.